target/start-mailserver.sh

```diff
@@ -216,8 +246,13 @@ else
   echo "==> Warning: 'SASL_PASSWD' is not provided. /etc/postfix/sasl_passwd not created."
 fi
 
-echo "Fixing permissions"
-chown -R 5000:5000 /var/mail
+# Fix permissions, but skip this if 3 levels deep the user id is already set
```

Comment:
There's a pending PR from @Josef-Friedrich https://github.com/tomav/docker-mailserver/pull/193
You should discuss the implementation to only keep one.
Thanks.